worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -991,7 +991,7 @@ class PushDataHandler extends BaseMessageHandler with Logging {
 
     val (messageMaster, messageSlave) =
       messageType match {
-        case Type.PUSH_DATA | Type.PUSH_DATA_HAND_SHAKE =>
```

Comment:
duplicate match

https://github.com/apache/incubator-celeborn/blob/7d7107d607b853ab61b8299f83de2e3dacfa7e3c/worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala#L994-L1000

Reply:
ok, the duplicate match can be removed